src/App.js

```diff
@@ -1,10 +1,26 @@
-
+import styled from "styled-components";
+import Footer from "./components/Footer";
+import MainSection from "./components/MainSection";
+import Navbar from "./components/Navbar";
 
 function App() {
   return (
-    <div className="App">
-    </div>
+    <Wrapper>
+      <Navbar />
+      <MainSection />
+      <Footer />
+    </Wrapper>
   );
 }
 
 export default App;
+
+const Wrapper = styled.div`
```

Comment:
dużo zależy od podejścia zespołu do realizowania projektu, ale w tych co brałem udział była zasada, że style przechowywane są w osobnym pliku - coś jak HTML i CSS, czyli mamy jeden plik odpowiedzialny za strukturę a drugi za style. Jeśli chodzi o nazewnictwo to często była to taka sama nazwa jak nazwa komponentu z dopiskiem .styles.js, czyli w tym przypadku 'App.styles.js'. I z tego pliku są importowane konkretne styled-componenty

Reply:
to generalnie dotyczy każego komponentu jaki masz, ale jak wspomniałem - możliwe jest to, że jakiś zespół będzie chciał w taki sposób pracować, chociaż z moich obserwacji raczej się od czegoś takiego odchodzi i tworzy osobny plik